src/db/models/users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../main');
+
+const Users = sequelize.define('User', {
```

Comment:
Consider using a more descriptive variable name for 'Users', such as 'UserModel' to better represent its purpose.